docs/discussion/TermComments.md

```diff
@@ -0,0 +1,25 @@
+# Term Comments
+
+## Adding Comments to Terms
+
+Comments may be added to ontology terms to further explain their intended usage.
+
+Wherever possible, we strive to use standard language for similar types of comments and suggest alternative terms to use. 
+
+Some examples of comments, and standard language for their usage, are:
+
+## Do Not Annotate
```

Comment:
Here make a strong note that this is not universally done, and even controversial. Point out this is how Gene Ontology does it, and that other ontologies do not use these kinds of annotations.